app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt

```diff
@@ -689,6 +691,36 @@ class PinPasswordActivityTest {
     }
   }
 
+  @Test
+  fun testPinPasswordActivity_checkKeyboardType_clickShowHidePassword_keyboardTypeIsSame() {
```

Comment:
rename to `testPinPasswordActivity_checkInputType_clickShowHidePassword_inputTypeIsSame` as in the core we are matching the input type

Reply:
Done